source/tutorials/table-filtering.rst

```diff
@@ -0,0 +1,88 @@
+
+.. command-block::
+    qiime feature-table filter-features --i-table table.qza --o-filtered-table filtered-table --p-min-samples 2
+
+Similarly, samples that contain only a few features could be filtered from a feature table as follows.
+
+.. command-block::
+    qiime feature-table filter-samples --i-table table.qza --o-filtered-table filtered-table --p-min-features 10
+
+Both of these methods can also be applied to filter contingent on the maximum number of features or samples, using the ``--p-max-features`` and ``--p-max-samples`` parameters.
+
+Identifier-based filtering
+--------------------------
+
+Identifier-based filtering is used to retain only a user-specified list of samples or features. In this case, the user will provide a tab-separated text file as input with the ``--m-sample-metadata-file`` or ``--m-feature-metadata-file`` parameter (for ``filter-samples`` or ``filter-features``, respectively) where the first column in the file contains that identifiers that should be retained, and the first row contains headers or names for each column. Only the first column in this file will be used, so there are no requirements on subsequent columns (if any are present). As a result, sample or feature metadata files can be used with this parameter. Identifier-based filtering can be applied as follows to remove samples from a feature table.
+
+First, we'll write a header line and two sample identifiers to a new file called ``samples-to-keep.tsv``. (If you already have a tsv file containing a header line and the identifiers of the samples that you want to keep, you can skip this step. Otherwise, in practice, you'd probably create this file in a text editor, not on the command line as is being done here.)
+
+.. command-block::
+    echo Identifier > samples-to-keep.tsv
+    echo L1S8 >> samples-to-keep.tsv
+    echo L1S105 >> samples-to-keep.tsv
+
+Then, we'll call the ``filter-samples`` method with the parameter ``--m-sample-metadata-file samples-to-keep.tsv``. The resulting table will contain only the two samples whose identifiers are listed in ``samples-to-keep.tsv``.
+
+.. command-block::
+    qiime feature-table filter-samples --i-table table.qza --m-sample-metadata-file samples-to-keep.tsv --o-filtered-table filtered-table
+
+Metadata-based filtering
+------------------------
+
+Metadata-based filtering is similar to identifier-based filtering, except that the list of identifiers to keep is determined based on metadata rather than being provided by the user directly. This is achieved using the ``--m-sample-metadata-file`` or ``--m-feature-metadata-file`` parameter (for ``filter-samples`` or ``filter-features``, respectively) and the ``--p-where`` parameter. The user provides a description of the samples that should be retained based on their metadata using ``--p-where``, where the syntax for this description is the SQLite where-clause syntax.
```

Comment:
Also I think we should link out `SQLite where-clause` to something like: https://www.tutorialspoint.com/sqlite/sqlite_where_clause.htm (a more canonical resource would be nice, but the SQLite homepage was a grammar definition, which isn't user-friendly)